modules/nixos/tty.nix

```diff
@@ -1,12 +1,20 @@
 { config, lib, ... }:
 let
   inherit (config.catppuccin) sources;
-  cfg = config.console.catppuccin;
+  cfg = config.catppuccin.tty;
```

Comment:
again, I think `console` is better because it's the namespace NixOS uses. From a user's perspective, most don't know that the name of the catppuccin port is `tty`, but they may have changed a few `console.*` options, so it follows that they expect the catppuccin option to be named `console`.

Reply:
same as https://github.com/catppuccin/nix/pull/371#discussion_r1855162178